socket.c

```diff
@@ -335,16 +383,69 @@ void *telnet_listening_thread(void *args)
 	pthread_attr_setdetachstate(&attr, PTHREAD_CREATE_DETACHED);
 
 	// Set thread name
-	prctl(PR_SET_NAME,"telnet listener",0,0,0);
+	prctl(PR_SET_NAME,"telnet-v4",0,0,0);
```

Comment:
Should use `IPv4` to distinguish this from appearing to be `telnet version 4 API`.